README.md

```diff
@@ -42,10 +42,40 @@ Example: *mainnet/Lisk/images/application/lisk.svg*
 
 ## Contributing
 
+You can create a pull request to propose changes you've made to a fork of app-registry.
+
 To contribute to the application registry with the metadata for your blockchain application, please submit a *Pull Request* with the necessary changes. Before submitting a PR, please read the [Repository Structure](#repository-structure) section.
```

Comment:
The entire contents of `Creating a pull request from a fork` can be moved to `./docs/creating-pull-request-from-a-fork.md`.
```suggestion
To contribute to the application registry with the metadata for your blockchain application, please [fork the repository and submit a *Pull Request*](./docs/creating-pull-request-from-a-fork.md) with the necessary changes. Before submitting a PR, please read the [Repository Structure](#repository-structure) section.
```